### DISCUSSION TITLE
希望作者能添加音标显示,开发者在不会读的时候能快速查看有音标后知道怎么拼读

### DISCUSSION THREAD

**Discussion by wjmboss** [YiiGuxing/TranslationPlugin#6214] [Ideas]
希望作者能添加音标显示,开发者在不会读的时候能快速查看有音标后知道怎么拼读,有发音可以听了非常棒,不过工作时不方便听或者机器没有喇叭,该插件作者已经做得非常好,辛苦作者的付出,该插件是广大开发者的福音(英语好的略过)

**Comment by YiiGuxing** [comment]
有没有音标是由翻译服务功能决定的。比如谷歌翻译是提供音标的，只是目前国内访问不了，需要梯子。以前有道翻译也是提供的，然而后面它又不提供了，看样子应该不会恢复提供了。。。。唉~目前我也没有办法了。。。

感觉您的肯定与支持！
